x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/hooks.ts

```diff
@@ -35,46 +32,35 @@ export function useGetAllAssignedEventFilters(
 }
 
 export function useSearchAssignedEventFilters(
```

Comment:
This function name is misleading. This function is not only searching for the explicitly `assigned event filters` but also for the `global ones`.  In that sense this should be called `useSearchAssignedAndGlobalEventFilters`

Reply:
I'm ok to change this name if is misleading but at the end, all the global artifacts are assigned ones, the difference is they are assigned to all the policies but at the end are `assigned`.

Reply:
sounds good to me to keep it like this then.